tests/unit/task/test_runnable.py

```diff
@@ -23,12 +30,28 @@ class MockConfig:
     threads: int = 1
     target_name: str = "mock_config_target_name"
 
+    def get_default_selector_name(self):
+        return None
+
 
 class MockRunnableTask(GraphRunnableTask):
-    def __init__(self, exception_class: Exception = Exception):
+    def __init__(
+        self,
+        exception_class: Exception = Exception,
+        nodes: Optional[List[MockNode]] = None,
+        edges: Optional[List[Tuple[str, str]]] = None,
+    ):
+        nodes = nodes or []
+        edges = edges or []
+
         self.forced_exception_class = exception_class
         self.did_cancel: bool = False
         super().__init__(args=MockArgs(), config=MockConfig(), manifest=None)
+        self.manifest = make_manifest(nodes=nodes)
```

Comment:
This feels like we are reimplementing the task logic in the test again. I view unit test as documenting behavior in this case,  so the only thing we need to test at the runnable task is: are we using the correct arguments used when calling `get_graph_queue`?